module/os/linux/zfs/abd_os.c

```diff
@@ -284,7 +275,7 @@ abd_unmark_zfs_page(struct page *page)
  * reclaim or compaction.  When necessary this function will degenerate to
  * allocating individual pages and allowing reclaim to satisfy allocations.
  */
-static void
+void
 abd_alloc_pages(abd_t *abd, size_t size)
```

Comment:
I think a better name for this would be `abd_alloc_chunks`, since it's also used on FreeBSD where it isn't necessarily allocating pages.  (also abd_free_pages->chunks)

Reply:
Sounds good. I will update this.